user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add 
+     them to their own calendar views, and copy related events to their own calendars.
+
+
+.. _admin-capabilities-sys-calendar-events: 
+
+Manage System Calendar Events
+-----------------------------
+
+If this capability has been set to *"System"*, the user create, edit, and delete events in system-wide calendars, which 
+are described in more detail in the <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+System calendar events are usually managed by the company managers and HRs.
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add them to their 
+     own calendar views, and copy related events to their own calendars.
+
+
+.. _admin-capabilities-sys-calendars: 
+
+Manage System Calendars
+-----------------------
+
+
+If this capability has been set to *"System"*, the users can 
+:ref:`create <user-guide-calendars-system>` and :ref:`manage <user-guide-calendars-manage>` system-wide calendars.
+
+System-wide calendars are usually created and managed by system administrators and top managers.
+
+
+.. _admin-capabilities-passwords:
+ 
+Manage Users' Passwords
+-----------------------
+
+If the capability is set to *"System"*, the user can change passwords of other users. Usually, this is only done
+by system administrators when :ref:`creating or editing a user record <user-management-users>`. 
+
+.. hint::
+
+    This capability does not influence a user's ability to :ref:`edit their own 
+    password <user-guide-getting-started-profile-password>` from the *"My User"* page.
+
+
+.. _admin-capabilities-merge:
+
+Merge Entities
+--------------
+
+If the capability is set to *"System"*, the user can :ref:`merge <user-guide-ui-components-grids-delete-merge>` 
+several records of the same entity. By default the only entity, records of which can be merged is 
+:ref:accounts <user-guide-accounts>, but if other mergebale entities are added the setting will be applied. 
+
+By default, this capability should be set to *"System"* for  usually used by sales representatives, 
+marketing associates, and other employees working with accounts.
+
+
+.. _admin-capabilities-outlook:
+
+Outlook Integration
+-------------------
+
+This capability is meaningful for Enterprise users only. If the capability is set to *"System"*, the user can 
+:ref:`download <user-guide-getting-started-my-user-outlook>` the latest version of the OroCRM add-in for 
+Outlook. 
+
+If :ref:`OroCRM is synchronized with Outlook <user-guide-synch-outlook>`, you can use the add-in to synchronize 
+contacts, tasks, and calendar events between OroCRM and your Outlook account. You can also associate emails to accounts, 
+contacts, opportunities, and cases, as well as create leads, opportunities, and cases in OroCRM based on emails from 
```

Comment:
Edit: contacts, opportunities, and cases, as well as create leads, opportunities, and cases in OroCRM that are based on emails from